### PR TITLE
First version of EmojiPicker component

### PR DESCRIPTION
**Pull request by arthur-caillaud**

### CODE REVIEW

**Comment by arthur-caillaud** [comment]
Starts to solve #34

**Comment by arthur-caillaud** [comment]
capture d ecran 2018-07-31 a 08 29 51

**Comment by arthur-caillaud** [comment]
Emojis picked are not added to the chatbar. Enhancement needs to be done to have Feed and EmojiPicker fully linked. Will be solved in another PR.

File: public/index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Aprico</title>
```
Comment (larmx) [inline public/index.html]: Awesome

Comment (arthur-caillaud) [reply public/index.html]: Waw

**Pull request merged**